test/e2e/util.go

```diff
@@ -235,7 +244,7 @@ func lookupCNAMEMiekg(host, server string) (string, error) {
 		return "", err
 	}
 	if len(r.Answer) == 0 {
-		return "", fmt.Errorf("not found")
+		return "", fmt.Errorf("No results for the host:%s in nameServer: %s ", host, server)
```

Comment:
I think this is a bit of an overkill as error. Here is what we are getting if this error is used:
```
 operator_test.go:221: Looking for DNS record in nameserver: ns-596.awsdns-10.net
 operator_test.go:227: Waiting for DNS record: myroute.1639073290.3c8d057.example-test.info, error: No results for the host :myroute.1639073290.3c8d057.example-test.info. in nameServer : ns-596.awsdns-10.net
```
Note the redundant hostname and nameserver. They are passed to the function, so the caller always knows them and can log them if there is a need, that's what happened here.

Compare the previous example to this:
```
operator_test.go:221: Looking for DNS record in nameserver: ns-596.awsdns-10.net
operator_test.go:227: Waiting for DNS record: myroute.1639073290.3c8d057.example-test.info, error: not found
```